db/seeds.rb

```diff
@@ -12,6 +12,7 @@
   Journeys::Configuration.create!(routing_name: Journeys::GetATeacherRelocationPayment::ROUTING_NAME, current_academic_year: AcademicYear.current)
   Journeys::Configuration.create!(routing_name: Journeys::FurtherEducationPayments::ROUTING_NAME, current_academic_year: AcademicYear.current)
   Journeys::Configuration.create!(routing_name: Journeys::EarlyYearsPayment::Provider::ROUTING_NAME, current_academic_year: AcademicYear.current)
+  Journeys::Configuration.create!(routing_name: Journeys::EarlyYearsPayment::Start::ROUTING_NAME, current_academic_year: AcademicYear.current)
```

Comment:
can we rename `Start` to `Claimant`? I don't think people will refer to it as the "start" journey and more likely call it the "claimant" journey

Reply:
even so, I don't think `Start` is the best name - it's all I could come up with - there must be something better